build.gradle

```diff
@@ -87,65 +87,65 @@ configurations {
+    implementation'org.glassfish.jersey.inject:jersey-hk2:2.26'
+    implementation'org.glassfish.hk2:guice-bridge:2.5.0'
 
     // TODO: migrate to the Jersey 2 client
-    compile 'com.sun.jersey:jersey-client:1.19.4'
+    implementation'com.sun.jersey:jersey-client:1.19.4'
 
-    compile 'com.fasterxml.jackson.core:jackson-core:2.9.3'
-    compile 'com.fasterxml.jackson.core:jackson-annotations:2.9.3'
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.9.3'
-    compile 'com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.9.3'
-    compile 'com.fasterxml.jackson.datatype:jackson-datatype-guava:2.9.3'
+    implementation'com.fasterxml.jackson.core:jackson-core:2.9.3'
+    implementation'com.fasterxml.jackson.core:jackson-annotations:2.9.3'
+    implementation'com.fasterxml.jackson.core:jackson-databind:2.9.3'
+    implementation'com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.9.3'
+    implementation'com.fasterxml.jackson.datatype:jackson-datatype-guava:2.9.3'
 
-    compile 'com.google.inject:guice:4.1.0'
-    compile 'com.google.inject.extensions:guice-servlet:4.1.0'
-    compile 'com.google.guava:guava:23.5-jre'
-    compile 'com.diffplug.durian:durian:3.4.0'
-    compile 'joda-time:joda-time:2.9.9'
+    implementation'com.google.inject:guice:4.1.0'
+    implementation'com.google.inject.extensions:guice-servlet:4.1.0'
+    implementation'com.google.guava:guava:23.5-jre'
+    implementation'com.diffplug.durian:durian:3.4.0'
+    implementation'joda-time:joda-time:2.9.9'
 
-    compile 'ch.qos.logback:logback-core:1.2.3'
-    compile 'ch.qos.logback:logback-classic:1.2.3'
-    compile 'net.logstash.logback:logstash-logback-encoder:4.11'
-    compile 'org.slf4j:jul-to-slf4j:1.7.25'
-    compile 'org.slf4j:log4j-over-slf4j:1.7.25'
-    compile 'org.slf4j:jcl-over-slf4j:1.7.25'
+    implementation'ch.qos.logback:logback-core:1.2.3'
+    implementation'ch.qos.logback:logback-classic:1.2.3'
+    implementation'net.logstash.logback:logstash-logback-encoder:4.11'
+    implementation'org.slf4j:jul-to-slf4j:1.7.25'
+    implementation'org.slf4j:log4j-over-slf4j:1.7.25'
+    implementation'org.slf4j:jcl-over-slf4j:1.7.25'
 
-    compile 'de.danielbechler:java-object-diff:0.94'
+    implementation'de.danielbechler:java-object-diff:0.94'
 
-    compile 'com.amazonaws:aws-java-sdk:1.11.248'
+    implementation'com.amazonaws:aws-java-sdk:1.11.248'
 
     compile('org.apache.curator:curator-recipes:5.3.0')
     compile('org.apache.zookeeper:zookeeper:3.8.0')
     compile('org.apache.curator:curator-test:5.3.0')
 
-    compile 'org.apache.commons:commons-lang3:3.7'
-    compile group: 'org.apache.commons', name: 'commons-text', version: '1.10.0'
-    compile 'commons-io:commons-io:2.6'
+    implementation'org.apache.commons:commons-lang3:3.7'
+    implementation group: 'org.apache.commons', name: 'commons-text', version: '1.10.0'
+    implementation'commons-io:commons-io:2.6'
     compile('com.github.rholder:guava-retrying:2.0.0') {
```

Comment:
Is there a reason this one wasn't changed?

Reply:
Updated compile() to implementation()